vald/values.yaml

```diff
@@ -232,6 +232,15 @@ gateway:
       target: agent
       image: busybox
       sleepDuration: 2
+  env:
+    - name: MY_NODE_NAME
+      valueFrom:
+        fieldRef:
+          fieldPath: spec.nodeName
```

Comment:
setting node name is optional for gateway, if we define nodename for discovery we should set component kind as DaemonSet, but gateway needs performance we can skip this setting.
node name config for gateway will be used in node level gateway

Reply:
I've just commented out these settings. thanks :+1: